lms/static/js/my_courses_dropdown.js

```diff
@@ -54,22 +54,19 @@ $(document).ready(function() {
             event.preventDefault();
         }
 
-    // if up arrow key pressed or shift+tab
-        if (event.which == 38 || (event.which == 9 && event.shiftKey)) {
-      // if first item go to last
+        // if up arrow key pressed or shift+tab else down key or tab is pressed
+        if (event.which === 38 || (event.which === 9 && event.shiftKey)) {
```

Comment:
Please create named variables for all of these key codes.  You can remove the comment after that since it will be readable.

e.g. `var upArrowKey = 38;`